### PR TITLE
image

### PR DESCRIPTION
**Pull request by RaghavendraaReddy**
- review please
- [Code Sandbox Live](https://5yeomv.csb.app/image.html)

### CODE REVIEW

File: image.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    color: black;
+    text-decoration: none;
+}
+.footer{
+    display: flex;
+    gap:1rem;
+    justify-content: center;
+    align-items: center;
+    background-color: black;
+    padding: 40px 40px;
```
Comment (HarshnaNHaswani) [inline image.css:29]: Keep repeating code separate

File: image.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    margin-top: 10px;
+}
+.main{
+    grid-column: 2/4;
+    grid-row: 2/6;
+    padding: 20px;
+}
+*{
+    margin: 0;
+}
```
Comment (HarshnaNHaswani) [inline image.css:58]: keep resets separate

File: image.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    width: 200px;
+}
+.img-Resp{
+    width: 50%;
+    height: auto;
+}
+.img-circle{
+    height: 200px;
+    width: 200px;
+    border-radius: 100%;
```
Comment (HarshnaNHaswani) [inline image.css:73]: Avoid using px for large values

**Review by HarshnaNHaswani — COMMENTED** [review]
Few suggestions, rest looks good to me

File: image.css [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    font-size: 55px;
+    margin-left: 45px;
+}
+.a{
+    justify-content: flex-end;
+    display: flex;
+    align-items: center;
+    margin-left: auto;
+}
+.a1{
```
Comment (anupkrbid) [inline image.css:17]: class name can be better

File: image.html [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+            An image is a graphic representation of something.
+        </p>
+        <h3>
+            Basic image 250px/250px
+        </h3>
+        <img class="img-basic" src="https://m.media-amazon.com/images/M/MV5BNmMwZTJmN2QtMDY0Mi00ZjIwLWE3NGYtODA5OTkyYjY2ZGJlXkEyXkFqcGdeQXVyNzQwODEwNTM@._V1_.jpg" alt="pic">
+        <br>
+        <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=31px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=13.5px&lh=113%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%253Cimg%2520class%253D%2522img-basic%2522%2520src%253D%2522https%253A%252F%252Fm.media-amazon.com%252Fimages%252FM%252FMV5BNmMwZTJmN2QtMDY0Mi00ZjIwLWE3NGYtODA5OTkyYjY2ZGJlXkEyXkFqcGdeQXVyNzQwODEwNTM%2540._V1_.jpg%2522%2520alt%253D%2522pic%2522%253E%250A"
+            style="width: 1024px; height: 103px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (anupkrbid) [inline image.html:47]: avoid inline styles

File: image.html [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=31px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=13.5px&lh=113%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%253Cimg%2520class%253D%2522img-circle%2522%2520src%253D%2522https%253A%252F%252Fm.media-amazon.com%252Fimages%252FM%252FMV5BNmMwZTJmN2QtMDY0Mi00ZjIwLWE3NGYtODA5OTkyYjY2ZGJlXkEyXkFqcGdeQXVyNzQwODEwNTM%2540._V1_.jpg%2522%2520alt%253D%2522pic%2522%253E%250A"
+            style="width: 1024px; height: 103px; border:0; transform: scale(1); overflow:hidden;"
+            sandbox="allow-scripts allow-same-origin">
+            </iframe>
+    </div>
+</div>
+<footer class="footer">
+    <a href="#">GitHub</a>
+    <a href="">LinkedIn</a>
+    <a href="">Twitter</a>
```
Comment (anupkrbid) [inline image.html:74]: Don't keep blank href, at least add `#`

File: image.css [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+    margin: 0;
+}
+.main p,h3,div{
+    padding-top: 20px;
+}
+.img-basic{
+    height: 200px;
+    width: 200px;
+}
+.img-Resp{
```
Comment (anupkrbid) [inline image.css:66]: should be max-width

**Review by anupkrbid — COMMENTED** [review]
image is not responsive yet. plz, check that. Rest all okay

**Review by anupkrbid — APPROVED** [review]

**Pull request merged**